CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This changelog documents the changes between release versions.
 ## main
 Changes to be included in the next upcoming release
 
+- OpenTelemetry support added via support for NDC TypeScript SDK v1.3.0
```

Comment:
Should this changelog entry be under main here? Or a seperate V0 in this case? Or both? I can see the logic an any of the options, but just curious what your thoughts are.

Reply:
I just dumped it there because I'm about to move it under its own number when I do the release. Thinking about it properly, on the `v0` branch that heading should probably read `v0`, since you'll never see an unreleased change from `main` on this branch, by definition.

Reply:
Actually come to think of it, "unreleased" in a better general heading... instead of main or v0